target-app/Home/Views/HomeViewController.swift

```diff
@@ -8,14 +8,32 @@
 import UIKit
 import MapKit
 
-class HomeViewController: UIViewController {
+class HomeViewController: UIViewController, CLLocationManagerDelegate, MKMapViewDelegate {
     
     let mapView: MKMapView = {
         let map = MKMapView()
         map.overrideUserInterfaceStyle = .dark
         return map
     }()
     
+    let locationManager = CLLocationManager()
+    
+    func locationManager(_ manager: CLLocationManager, didUpdateLocations
+                         locations: [CLLocation]) {
```

Comment:
Pls check our Swift style guide, the indentation and line breaking sections specifically.
```suggestion
    func locationManager(
        _ manager: CLLocationManager, didUpdateLocations
        locations: [CLLocation]
    ) {
```